client/github_client_manager.go

```diff
@@ -0,0 +1,115 @@
+package client
```

Comment:
client/client_creator.go is renamed since it is responsible for token management.